curriculum/challenges/ukrainian/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-football-team-cards/63e95e39860dc5b01ebe9be0.md

```diff
@@ -9,14 +9,14 @@ dashedName: step-29
 
 Наступним кроком буде показ слова `(Captain)` біля гравця, якщо він вказаний як капітан команди.
```

Comment:
Looks like you mistakenly selected a file from a different folder other than english, Please try to change this file ?
![image](https://github.com/freeCodeCamp/freeCodeCamp/assets/61899866/dd663e07-6e44-4251-9123-a0f4b8717527)